TouchPortalAPI/sdk_tools.py

```diff
@@ -502,6 +508,15 @@ def _validateDefinition(entry, as_str=False):
     _printToErr(f"\nFinished validating '{name}'.\n")
     return res
 
+def generatePythonStruct(entry, name):
+    _printToErr("Generating Python struct from entry json...\n")
+    tp_to_py = TpToPy(entry)
+    try:
+        tp_to_py.writetoFile(name)
+        _printToErr(f"Saved generated Python struct to '{name}.py'\n")
+    except Exception as e:
+        _printToErr(f"Error: {e}")
+        return -1
```

Comment:
Why is this returning a value only on exception?

Reply:
Fixed